psiminer-cli/src/main/kotlin/config/LabelExtractorConfigs.kt

```diff
@@ -6,12 +6,12 @@ import labelextractor.MethodNameLabelExtractor
 import labelextractor.LabelExtractor
 
 @Serializable
-abstract class LabelExtractorConfig {
-    abstract fun createProblem(): LabelExtractor
+abstract class LabelExtractorConfig<T> {
+    abstract fun createProblem(): LabelExtractor<T>
 }
 
 @Serializable
 @SerialName("method name")
-class MethodNameLabelExtractorConfig : LabelExtractorConfig() {
-    override fun createProblem(): LabelExtractor = MethodNameLabelExtractor()
+class MethodNameLabelExtractorConfig : LabelExtractorConfig<String>() {
+    override fun createProblem(): LabelExtractor<String> = MethodNameLabelExtractor()
 }
```

Comment:
After using star-projection in the base class, there is no need to specify any direct types in derived classes. Therefore, this label extractor wouldn't be changed.